website/src/components/home/CanvasList.jsx

```diff
@@ -145,20 +152,22 @@ class CanvasList extends React.Component {
     const canvasList = [];
 
     Object.keys(this.props.canvases).forEach((canvasId, i) => {
-      canvasList.push(
-        <Box style={styles.models} key={i}>
-          <Link to="/canvas">
-            <Card onTouchTap={this.createClickHandler(canvasId)}>
-              <CardMedia
-                overlay={<CardHeader title={this.props.canvases[canvasId][RC.CANVAS_NAME]} />}
-                overlayContentStyle={styles.overlay}
-                >
-                {generateCanvasCode}
-              </CardMedia>
-            </Card>
-          </Link>
-        </Box>
-      )
+      if (userCanvasList[canvasId]) {
```

Comment:
this if statement isn't needed if the canvas has been removed from the list of canvases in the redux store